src/plugins/intel_cpu/tests/functional/single_layer_tests/augru_sequence.cpp

```diff
@@ -0,0 +1,391 @@
+    { { {{0, 11}, -1, {7, 11}},                     // #8. Dynamic shape 0
+        { {10, 2, 10}, {3, 4, 10}, {5, 5, 10}, {10, 2, 10}, {5, 5, 10} } },  // Target shapes
+      { {-1, 1, {8, 12}},                           // Dynamic shape 1
+        { {10, 1, 10}, {3, 1, 10}, {5, 1, 10}, {10, 1, 10}, {5, 1, 10} } },  // Target shapes
+      { {{0, 11}, -1, 1},                           // Dynamic shape 3
+        { {10, 2, 1}, {3, 4, 1}, {5, 5, 1}, {10, 2, 1}, {5, 5, 1} } },     // Target shapes
+      { {-1},                                       // Dynamic shape 2
+        { {10}, {3}, {5}, {10}, {5} } } }            // Target shapes
+};
+
+INSTANTIATE_TEST_SUITE_P(smoke_dynamic, AUGRUSequenceCPUTest,
+            ::testing::Combine(::testing::ValuesIn({dynamicShapes[0], dynamicShapes[1], dynamicShapes[2]}),
+                               ::testing::ValuesIn(mode),
+                               ::testing::ValuesIn(activations),
+                               ::testing::ValuesIn(clip),
+                               ::testing::ValuesIn(linearBeforeReset),
+                               ::testing::ValuesIn(direction),
+                               ::testing::ValuesIn(netPrecisions),
+                               ::testing::Values(cpuParams),
+                               ::testing::Values(std::map<std::string, std::string>{})),
+            AUGRUSequenceCPUTest::getTestCaseName);
+
+INSTANTIATE_TEST_SUITE_P(smoke_dynamic_BatchSizeOne, AUGRUSequenceCPUTest,
+            ::testing::Combine(::testing::ValuesIn({dynamicShapes[4]}),
+                               ::testing::ValuesIn(mode),
+                               ::testing::ValuesIn(activations),
+                               ::testing::ValuesIn(clip),
+                               ::testing::ValuesIn(linearBeforeReset),
+                               ::testing::ValuesIn(direction),
+                               ::testing::ValuesIn(netPrecisions),
+                               ::testing::Values(cpuParamsBatchSizeOne),
+                               ::testing::Values(std::map<std::string, std::string>{})),
+            AUGRUSequenceCPUTest::getTestCaseName);
+
+INSTANTIATE_TEST_SUITE_P(nightly_dynamic, AUGRUSequenceCPUTest,
+            ::testing::Combine(::testing::ValuesIn({dynamicShapes[5], dynamicShapes[8]}),
+                               ::testing::ValuesIn(mode),
+                               ::testing::ValuesIn(activations),
+                               ::testing::ValuesIn(clip),
+                               ::testing::ValuesIn(linearBeforeReset),
+                               ::testing::ValuesIn(direction),
+                               ::testing::ValuesIn(netPrecisions),
+                               ::testing::Values(cpuParams),
+                               ::testing::Values(std::map<std::string, std::string>{})),
+            AUGRUSequenceCPUTest::getTestCaseName);
+
+INSTANTIATE_TEST_SUITE_P(nightly_dynamic_bf16, AUGRUSequenceCPUTest,
```

Comment:
@nshchego What's the "nightly_dynamic_bf16" trying to cover in gru_sequence unit test? It is skipped actually.

I think, it must be a test for transformation, not for cpu rnn node, because the seq_lengths are not supported by cpu rnn node. 
(https://github.com/openvinotoolkit/openvino/blob/2d04f934868bd850a9f8857fcbb173a5e8b4381f/src/tests/ngraph_helpers/ngraph_functions/src/gru_cell.cpp#L41).

Reply:
Do you mean AUGRU tests fail or other RNN? Such failures should be highlighted by nightly builds.

Reply:
@nshchego 
The "_nightly_dynamic_bf16_" are skipped because there is a row _R"(.*(BF|bf)16.*(jit_avx(?!5)|jit_sse|ref).*)"_ in _src/plugins/intel_cpu/tests/functional/shared_tests_instances/skip_tests_config.cpp_.

That's why there is no error reported by CI and nightly builds.

However, if manually run it, there will be 4 cases fail in gru_sequence single layer test.

The failure indicates 2 problems -
1. We should not use _get_min_length_ for CONVERT_TO_TI_MAX_SEQ_LEN_CONST and PURE_SEQ. In case of dynamic input shape, as the test params used in _nightly_dynamic_bf16_, the seq_lengths are smaller than max_seq_length. It violates the definition of the two modes.
2. Even though it violates, there shouldn't produce difference in outputs.


So for this PR, which is to enable augru sequence with PURE_SEQ only, I will keep _nightly_dynamic_bf_16_ skipped.
And I will create a sperate PR/ticket to address the problem #2, as it is a common issue of both augrusequence and grusequence.
How do you think? And any suggestion for problem #1?

Reply:
@ceciliapeng2011 Not sure I fully understand details on #1 issue. Is it relates to ngraph transformations?

Anyway I am ok to resolve both issue in separate PR